CHARTER.md

```diff
@@ -0,0 +1,127 @@
+The Board shall have at least the following roles:
+
+ * Treasurer: Responsible for financial transactions and accounting.
+ * Secretary: Responsible for taking and publishing meeting notes and management of Voting membership.
+
+The Governing Board meets at least 4 times per year, publishes quarterly financial reports, and shares the content of their meetings with the MapLibre Community to the extent the Board deems appropriate.
+
+### Voting Members
+Voting Members are people who elect the Governing Board and who vote on changes to the MapLibre Charter.
+
+A precondition to becoming a Voting Member is that the person contributed in a non-trivial way or that they donated funds to the MapLibre Organization.
+
+If Voting Members are paid by a Third-party organization for their work on the MapLibre Projects, they shall declare publicly who their employer is. At most 10 percent of the Voting Members shall be employed by a single Third-party organization.
+
+Voting Members can nominate other Community members to receive Voting membership in acknowledgement of their contributions to the MapLibre Projects. The Voting Members then vote on granting the nominee a Voting membership. 
+
+Nominations for new Voting Members are open up to two weeks before the election of the Governing Board and the vote on granting Voting Membership to somebody takes place one week before the election of the Governing Board.
+
+### Governing Board Election
+Governing Board elections are announced at least four weeks in advance.
+
+Nomination process:
+
+ * A person can nominate themselves as a candidate for the Governing Board.
+ * Candidates have to register themselves up to 2 weeks before the election.
+
+Voting process:
+
+ * Governing Board elections take place every year.
+ * Each Voting Member ranks all nominees according to ranked-choice voting.
+ * The nominee with the largest number of votes gets appointed, the nominee with the second largest number of votes gets appointed, and so on.
+ * In case there are multiple nominees per Third-party organization, only the one with the largest number of votes gets appointed.
+ * In case of a tie where more nominees than remaining seats receive an equal number of votes, the vote shall be repeated for the remaining seats.
+
+### Terms of Governing Board Members
+Governing Board members will serve for a one-year term.
+
+### Governing Board Member Remuneration
+Governing Board members receive no remuneration by the MapLibre Organization for their services on the Board, other than reimbursement for the out-of-pocket expenses incurred when carrying out activities on behalf of the Governing Board. Such expenses must be pre-authorized by at least two other board members.
+### Conflicts of Interest
+The Governing Board develops a conflict of interest policy which details how Governing Board members can disclose conflicts of interest between their role as MapLibre Governing Board members and other professional and personal activities. 
```

Comment:
Per above, I think this can be lightweight, so this section would be more about awareness that conflicts will arise and the board members will be responsible about them. (Sorry, can’t draft something right now.)

Reply:
#100